01-lets_get_started/01.rb

```diff
@@ -5,24 +5,24 @@ def initialize(side)
     @gap_length = side - 2
   end
 
-  def first_last_line(side)
-    side.times { print '#' }
+  def first_last_line
+    @side.times { print '#' }
```

Comment:
Can't you do the same strategy for generating repeated string as you did in line 14?